packages/react/src/FormFieldCharacterCounter/FormFieldCharacterCounter.tsx

```diff
@@ -0,0 +1,22 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright Gemeente Amsterdam
+ */
+
+import clsx from 'clsx'
+import { forwardRef } from 'react'
+import type { ForwardedRef, HTMLAttributes } from 'react'
+
+export type FormFieldCharacterCounterProps = HTMLAttributes<HTMLDivElement> & {
+  label: string
```

Comment:
Please add `length` and `maxLength` props.

The component can render `${length} van ${maxLength} tekens` – we’ll worry about i18n later.